testutils/check-lang-uses-form/main_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"sort"
+	"strings"
+	"testing"
+)
+
+func TestReadChecks(t *testing.T) {
+	const checkFile = `
+Form1, // used by: java py
+FormNotChecked,
+// NestedCommentsIgnored, // used by: java
+FormNoComma // used by: java
+`
+	checks, err := readChecks(strings.NewReader(checkFile))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	wants := map[string][]string{
+		"java": {"Form1", "FormNoComma"},
+		"py":   {"Form1"},
+	}
+
+	for lang, want := range wants {
```

Comment:
consider inlining `wants`

Reply:
Do you feel strongly about this? Sometimes I do this and find the loop harder to read.

Reply:
Don't feel strongly, but IIUC it's the idiomatic way to do it. More compact.